integration-tests/helpers/index.js

```diff
@@ -11,9 +11,10 @@ const path = require('path')
 const assert = require('assert')
 const FakeAgent = require('./fake-agent')
 const id = require('../../packages/dd-trace/src/id')
-const { version } = require('../../package.json')
 const { getCappedRange } = require('../../packages/dd-trace/test/plugins/versions')
+const { BUN, withBun } = require('./bun')
 
+const sandboxRoot = path.join(os.tmpdir(), id().toString())
```

Comment:
Not that it matters from a technical perspective, but did you mean to add the id twice? (you're also adding it again further down in this file when using `sandboxRoot` to create the `folder` string)